docs/guide/persistence_cache.md

```diff
@@ -222,6 +222,21 @@ stash:
     The regular `php app/console cache:clear` command does not clear Redis persistence cache.
     To clear it, use the console command shipped with Redis: `redis-cli flushall`.
 
+!!! tip
+
+    If you use the Redis cache driver and encounter problems with memory consumption,
+    you can use the following (non-SiteAccess-aware) settings:
+
+    ``` yaml
+    ezpublish:
+        stash_cache:
+            igbinary: true
```

Comment:
This suggests that user should set both settings to true, maybe use `bool` or `true/false` instead of `true`.